ui/app/i18n/document-upload/locale_fr.json

```diff
@@ -0,0 +1,16 @@
+{ 
+    "VISIT_TYPE_OPHTHALMOLOGY": "Ophtalmologie",
+    "VISIT_TYPE_LABORATORY": "Laboratoire",
+    "VISIT_TYPE_GYNECOLOGY": "Gynécologie",
+    "VISIT_TYPE_MATERNITY": "Maternite",
+    "VISIT_TYPE_OPD": "Consultations Externes",
+    "VISIT_TYPE_CARDIOLOGY": "Cardiologie",
+    "VISIT_TYPE_ANC": "CPN",
+    "VISIT_TYPE_MINOR_SURGERY": "Petite Churigie",
+    "VISIT_TYPE_EMERGENCY": "Urgence",
+    "VISIT_TYPE_DELIVERY": "D'accouchement",
+    "VISIT_TYPE_DENTISTRY": "Dentisterie",
+    "VISIT_TYPE_RADIOLOGY": "Radiologie",
+    "VISIT_TYPE_PEDIATRIC": "Pediatrie",
```

Comment:
@prashantsable1991 Missing accent: Pédiatrie

Reply:
@tmvumbi2 added typos and missing accent. Please review once. Thanks.